### ISSUE TITLE
Bug: correlation_count in UsersController.php no longer divide by 2 with new correlations engine

### ISSUE THREAD

**Issue by github-germ** [MISP/MISP#8730]
### Expected behavior

The statistics correlations count should be double what is reported.

### Actual behavior

I don't believe the new correlations engine inserts 2 rows (ie. in both directions). Hence, I think the `/2` line can be removed in [UsersController.php](https://github.com/MISP/MISP/blob/v2.4.164/app/Controller/UsersController.php#L1935) in `__statisticsData` 
```
 $stats['correlation_count'] = $this->User->Event->Attribute->Correlations('recursive' => -1));
 $stats['correlation_count'] = $stats['correlation_count'] / 2;
```

### Steps to reproduce

https://MISP/users/statistics.json

### Version

2.4.164

### Operating System

RedHat

### Operating System version

8.6

### PHP version

7.4.30

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by iglocska** [comment]
Good point indeed!

**Comment by iglocska** [comment]
Will be included in the upcoming release via https://github.com/MISP/MISP/commit/249736937400c6f166924b425eb7e1238ce50c5a

**Comment by github-germ** [comment]
Thanks: fixed in 2.4.165.
